.github/workflows/sonar-backend.yaml

```diff
@@ -0,0 +1,64 @@
+name: SonarCloud Scan for backend
+on:
+  push:
+    branches:
+      - "main"
+  pull_request_target:
+    types: [ "opened", "edited", "reopened", "synchronize" ]
+    paths:
+      - "odd-platform-api/**"
+      - "odd-platform-api-contract/**"
```

Comment:
I don't see why we need to trigger the scan when something changed in the odd-platform-api-contract build.gradle

Reply:
since there are no non-generate files there's no points, removed